src/keria/app/delegating.py

```diff
@@ -164,5 +166,29 @@ def processPartialWitnessEscrow(self):
                     if not witnessed:
                         continue
                 print(f"Witness receipts complete, {pre} confirmed.")
+
+                hab = self.hby.habs[pre]
+                if isinstance(hab, habbing.GroupHab):
+                    phab = hab.mhab
+                elif hab.kever.sn > 0:
+                    phab = hab
+                elif self.proxy is not None:
+                    phab = self.proxy
+                else:
+                    raise kering.ValidationError("no proxy to send messages for delegation")
+
+                delpre = hab.kever.delegator  # get the delegator identifier
+                if delpre not in hab.kevers:
+                    raise kering.ValidationError(f"delegator {delpre} not found, unable to process delegation")
+                smids = []
+                sn = hab.kever.sner.num
+                evt = hab.makeOwnEvent(sn=sn)
+
+                exn, atc = delegateRequestExn(phab, delpre=pre, evt=bytes(evt), aids=smids)
+
+                notifier = Notifier(hby=self.hby)
+                handler = DelegateRequestHandler(hby=self.hby, notifier=notifier)
+                handler.handle(serder=exn)
+
```

Comment:
Hmm, couple of possible issues here. I'm not sure if this function will always run because identifiers can be witnessed or non-witnessed (direct mode). And if client1 delegates to client2 - who is receiving the notification?

There is a `loadHandlers` function already available in keripy that already loads `DelegateRequestHandler` - https://github.com/WebOfTrust/keripy/blob/development/src/keri/app/delegating.py#L196

We should just need to call this function in `agenting.py` of KERIA - https://github.com/WebOfTrust/keria/blob/development/src/keria/app/agenting.py#L337 - this is generally how handlers for routes are loaded so that it's easily loaded/unloaded and can be configured if needed, rather than keep in the code here

Reply:
updated
ref here: 1de4742fd8b6c59a2393025c880a578725af3f2a